src/lib/components/input-field.svelte

```diff
@@ -4,26 +4,44 @@
 	export let autofocus = false
 	export let disabled = false
 	export let pad = 0
+	export let label = ''
 
 	let inputField: HTMLInputElement
 </script>
 
-<!-- svelte-ignore a11y-autofocus -->
-<input
-	class="text-lg"
-	style={`padding-block: ${pad}px`}
-	type="text"
-	{disabled}
-	{autofocus}
-	{placeholder}
-	bind:value
-	bind:this={inputField}
-	on:keydown
-	on:keypress
-	on:keyup
-/>
+<div class="input-wrapper">
+	{#if label !== ''}
+		<span class="text-sm">{label}</span>
+	{/if}
```

Comment:
I think this should either be some named `slot` or the label should be potentially undefined `export let label: string | undefined = undefined`. I don't like having things set to empty strings and asserting to empty string.

Reply:
Cool, thank you. I'm not sure it should be a slot because it will never be anything other than a string that looks the exact same way, so having it as a prop protects the styling while being easier to declare in the code.
But I do agree regarding the empty string, I'll change that. 
Thanks!